2_process/src/functions.R

```diff
@@ -0,0 +1,25 @@
+process <- function(input){
+  
+  # Set output directory
+  project_output_dir <- "2_process/out"
```

Comment:
Similar comment in the `fetch` function: pull `project_output_dir` out of the body and make it an argument. These paths frequently change when working out a new pipeline. it also adds a bit of clarity for someone reviewing your top-level `_targets.R` file.